templates/default/cut_tiles.sh.erb

```diff
@@ -56,11 +56,6 @@ valhalla_build_tiles -c <%= node[:valhalla][:config] %> $(find <%= node[:valhall
 rm -rf *.bin
 set +e
 
-# package up the tiles and the admin and timezone dbs
-cur_tile_dir=<%= node[:valhalla][:base_dir] %>/tiles_${stamp}
-mkdir -p ${cur_tile_dir}
-mv <%= node[:valhalla][:tile_dir] %>/* ${cur_tile_dir}/
```

Comment:
cant move the stuff and then reference it later!? this should fix the issue with stats and empty connectivity stuff going to s3